### DISCUSSION TITLE
How to use subscription with the Prisma

### DISCUSSION THREAD

**Discussion by aramis-it** [MichalLytek/typegraphql-prisma#70] [Q&A] [answered]
"@prisma/client": "^2.14.0",
    "type-graphql": "^1.1.1",
    "typegraphql-prisma": "^0.9.4",

I don't understand documentation clearly. Could you show me some examples? Existing examples seem outdated.

1. How to subscribe client to a topic when client connected.
2. How to push data to clients.

**Comment by MichalLytek** [comment, marked as answer]
Prisma 2 doesn't support subscriptions yet like the Prisma 1
